Kernel_23/doc/Kernel_23/CGAL/Projection_traits_xy_3.h

```diff
@@ -21,18 +21,12 @@ constructions or if `K` is a `CGAL::Filtered_kernel` (such as for
 `CGAL::Exact_predicates_inexact_constructions_kernel`), this class automatically
 provides exact predicates.
 
-\cgalModels The class is a model of several 2D triangulation traits class concepts,
+\cgalModelsBare{The class is a model of several 2D triangulation traits class concepts\,
```

Comment:
I understand that you usually models only one concept but that a concept have several models.

Reply:
The situation wit the `cgalModels` is that here the models don't have template like constrict when referring to it , whilst `cgalHasModels` does have it. For this with `cgalHasModels` the choice has been made to use a `begin` / `end` construct and with `cgalModels` a construct with the multi arguments was made. A few cases there were some "bare" constructs that needed some tricks, sometimes there was a `,` in the field and this could be prevented by means of the `\,` construct on other places multiple lines were necessary, hence the `\n`.  This all could be overcome when the `ALIASES` had had the possibility to specify a separator, for this I did recently create the proposed pull request https://github.com/doxygen/doxygen/pull/10179
Maybe we normally should use the `cgalModels` construct but in the case off `cgalModelsBare` use a `begin` / `end` construct as well to make it all more transparent.